tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -577,4 +592,15 @@ private function getProductQuantitiesByReference(string $orderReference, string
 
         return $productQuantities;
     }
+
+    private function assertOrderPropertiesEquals(Order $order, array $data): bool
+    {
+        foreach (array_keys($data) as $property) {
+            if (!property_exists($order, $property) || $data[$property] !== $order->$property) {
+                return false;
```

Comment:
Instead of returning tru, and using `Assert::assertTrue` Maybe you can throw a `RuntimeException` with an explicit message? (like which field is incorrect, expected value and found result...)
This will make debugging easier when something wrongs happens, instead of one boolean for the whole object

Reply:
I moved the behat test to https://github.com/sowbiba/PrestaShop/pull/6 to unblock 1.7.7 release